website/pages/[locale]/index.mdx

```diff
@@ -37,10 +36,11 @@ ${t('index.supportedNetworks.description')}
 <SupportedNetworks />
 ${t('index.supportedNetworks.footer').replace('{0}', `[${t('index.supportedNetworks.title')}](/developing/supported-networks/)`)}`
   const mdx = await buildDynamicMDX(rawMdx, { codeHighlight: false })
+  const { pageMap } = await import(`.next/static/chunks/nextra-page-map-${locale}.mjs`)
```

Comment:
This seems... dangerous. 😂

Reply:
why? this is only generated at build time

Reply:
this will not change, Nextra 3 under the hood saves traversed `pageMap` to according files